spinta/backends/mongo.py

```diff
@@ -0,0 +1,102 @@
+import contextlib
+
+from spinta.commands import load, prepare, migrate, check, push, get, getall, wipe
+from spinta.components import Context, BackendConfig, Manifest, Model
+from spinta.backends import Backend
+
+
+class Mongo(Backend):
+    # Instance of this class will be created when application starts. First if
+    # will be loaded from configuration using `load` command, then it will be
+    # prepared from manifest declarations using `prepare` command.
+    #
+    # Backend also must have a `transaction` method which must return read or
+    # write transaction object containing an active `connection` to database.
+    metadata = {
+        'name': 'mongo',
+        'properties': {
+            'dsn': {'type': 'string', 'required': True},
+        },
+    }
```

Comment:
`metadata` is an leftover from the past and no longer needed or used. You can convert it to:
```suggestion
    schema = {
        'dsn': {'type': 'string', 'required': True},
    }
```

Reply:
currently without `metadata` I am getting this error:

```
Traceback (most recent call last):
  File "/Users/justas/work/tilaajavastuu/spinta/spinta/testing/pytest.py", line 25, in context
    load_commands(config.get('commands', 'modules', cast=list))
  File "/Users/justas/work/tilaajavastuu/spinta/spinta/utils/commands.py", line 19, in load_commands
    module = importlib.import_module(module_path)
  File "/Users/justas/.pyenv/versions/3.7.1/Python.framework/Versions/3.7/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/justas/work/tilaajavastuu/spinta/spinta/backends/mongo.py", line 13, in <module>
    class Mongo(Backend):
  File "/Users/justas/work/tilaajavastuu/spinta/spinta/types/__init__.py", line 41, in __new__
    assert 'metadata' in attrs
AssertionError
```

It's implemented using `PostgreSQL` backend as example.

Reply:
Ok, then leave it for now.